SwiftDeck/README.md

```diff
@@ -0,0 +1,60 @@
+
+# SwiftDeck
+> Swift wrapper around deck.gl
```

Comment:
Nit: We usually use the `>` markdown style for "warnings" in this case we would call out that this is an experimental  module.